src/mock/responseData.ts

```diff
@@ -0,0 +1,40 @@
+import { IResponseDataProps, ResponseStatus } from 'context';
+import { StudentArray } from './studentData';
+
+export const ResponseRomanShashkov: IResponseDataProps = {
```

Comment:
Is it necessary to separate responses into different objects?
I think it is better to join them into array: IResponseDataProps[]